spyder/config/main.py

```diff
@@ -205,7 +205,7 @@
               'tab_always_indent': False,
               'intelligent_backspace': True,
               'automatic_completions': True,
-              'automatic_completions_after_chars': 3,
+              'automatic_completions_after_chars': 1,

```

Comment:
@ccordoba12 What do you think about this one, leaving it in 1 by default may cause regressions, such as #10134

Reply:
based on my testing, the more important value for performance is the 300ms, but I'm willing to change this if you disagree.